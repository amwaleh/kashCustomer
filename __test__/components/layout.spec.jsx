import * as React from "react";
import { shallow } from "enzyme";
import Layout from '../../components/layout'


  describe("Layout", () => {
    it("should render without throwing an error", function () {
      const wrapper = shallow(<Layout />);
      expect(wrapper.debug()).toMatchSnapshot();
    });
  });
