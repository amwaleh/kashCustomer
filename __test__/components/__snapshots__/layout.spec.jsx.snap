// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render without throwing an error 1`] = `
"<div>
  <Head>
    <title>
      WooCom
    </title>
    <meta charSet=\\"utf-8\\" />
    <meta name=\\"viewport\\" content=\\"initial-scale=1.0, width=device-width\\" />
  </Head>
  <header />
  <Grid>
    <GridRow className=\\"topNavRow\\" color=\\"teal\\">
      <Menu fluid={true} color=\\"purple\\">
        <MenuItem icon=\\"power\\" name=\\"log out\\" position=\\"right\\" onClick={[Function]} />
      </Menu>
    </GridRow>
    <GridRow className=\\"sidebarNav\\">
      <GridColumn as={[Function: Menu]} width={2} color=\\"teal\\">
        <div className=\\"spacer\\">
          <Header as=\\"h2\\" icon={true}>
            <Icon name=\\"user circle\\" inverted={true} as=\\"i\\" />
            Customer Dashboard
            <HeaderSubheader>
              Manage customer Details.
            </HeaderSubheader>
          </Header>
        </div>
        <MenuItem icon=\\"home\\" name=\\"home\\" link={true} href=\\"/home\\" />
        <MenuItem icon=\\"add user\\" name=\\"add customer\\" link={true} href=\\"/customer\\" />
      </GridColumn>
      <GridColumn width={12}>
         
      </GridColumn>
    </GridRow>
  </Grid>
</div>"
`;
